scripts/build-windows-installer.ps1

```
@@ -556,20 +630,16 @@ if not findings or not any(finding.analyzer == 'YARA' for finding in findings):
-Invoke-CheckedProcess (Join-Path $validationRuntime 'python.exe') @('-I', '-c', $dependencyCheck)
+Invoke-CheckedProcess $validationPython @('-I', '-c', $dependencyCheck)
 
 $siteZip = Join-Path $payload "site-packages.zip"
-Write-ZipFromDirectory $sitePackages $siteZip
+Write-ZipFromDirectory $sitePackages $siteZip $validationPython $sourceDateEpoch $reproducibilityRoot
 
 $launcher = Join-Path $payload "defenseclaw-launcher.exe"
-Invoke-CheckedProcess "go" @(
-    "build", "-ldflags", "-s -w", "-o", $launcher, "./cmd/defenseclaw-launcher"
-)
+Build-VerifiedGoBinary $launcher './cmd/defenseclaw-launcher' "-s -w -buildid=defenseclaw-launcher-$sourceCommit" $reproducibilityRoot
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Build-VerifiedGoBinary' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)